client/Windows/wf_defaults.c

```diff
@@ -0,0 +1,112 @@
+#include <Windows.h>
+#include <wincred.h>
+#include <stdio.h>
+#include <malloc.h>
+
+#pragma warning(disable : 4706 4094) // assignment within conditional expression
+
+#define FreeRDP_ServerHostname (20)
+#define FreeRDP_Username (21)
+#define FreeRDP_Password (22)
+#define FreeRDP_GatewayHostname (1986)
+#define FreeRDP_GatewayUsername (1987)
+#define FreeRDP_GatewayPassword (1988)
+
+typedef struct rdp_settings rdpSettings;
+
+EXTERN_C_START
+
+const char* freerdp_settings_get_string(const rdpSettings* settings, size_t id);
```

Comment:
why not simply `#include <freerdp/settings.h>`?

Reply:
good point!